packages/core/src/components/DataItem/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Text from '../Text';
+
+function DataItem({
+  label,
+  link,
+  floatLinkRight,
+  inline,
+  children,
+  cx,
+  styles,
+}: {
+  label: string;
+  link?: NonNullable<React.ReactNode>;
+  floatLinkRight?: boolean;
```

Comment:
instead of `right` - please use `end`. maybe just `endAlign` to be consistent with naming?

looking at the comps, i would do something like:

```jsx
<div className={cx(inline && styles.inline)}>
  <div className={cx(styles.wrapper, rightAlign && styles.wrapper_rightAlign)}>
    <Spacing right={1}>
      <Text small muted>
        {label}
      </Text>
    </Spacing>

    {link && (
      <Text small muted>
        {link}
      </Text>
    )}
  </div>

  <Text>{children}</Text>
</div>


export default withStyles(() => ({
  inline: {
    display: 'inline-block',
  },

  wrapper: {
    display: 'flex',
  },

  wrapper_rightAlign: {
    justifyContent: 'space-between',
  },
}))(DataItem);
```

this would avoid some funky stuff that sometimes happens with floats and not clearing them

Reply:
Good catch, thanks!